### ISSUE TITLE
Meyersound Galileo - Galaxy

### ISSUE THREAD

**Issue by pyrahc** [bitfocus/companion-module-meyersound-galaxy#1]
This is an audio system processor from Meyersound.

I would like to be able to control mutes, groups of mutes, and recalling snapshots.

here is a link to the control pdf

https://meyersound.com/download/galaxy-programming-guide/?wpdmdl=99216&refresh=662b54d7296271714115799&ind=1634665762893&filename=galileo-galaxy_pg_b4.pdf

and to the product

https://meyersound.com/product/galileo-galaxy/

If it is possible to have toggle buttons - mute/unmute.

also in the pdf there is "get control point value" which returns the mute status. so using this would be great to reflect the staus on the button, even if the mute button on the processor is pressed

any questions please don't hesitate to contact

**Comment by mattsteuart** [comment]
I would second this one. absolutely.

**Comment by jomixlaf** [comment]
Hi, I developed the module to control the Galaxy! I've added a bunch of features already; it's not yet ready, but it does work!

- Mute input output
- Set gain input, output, matrix
- Programmable fade in, out and matrix, 
- many variables. 
- Change AVB input Stream. 
- more....

**Comment by endlessrhoads** [comment]
I'm curious about the status of this. I would love to be able to use it. Specifically, I would like to be able to do snapshot changes.

**Comment by jomixlaf** [comment]
I made significant progress, implementing almost 95% of the entire functionality. Now, I am focusing on consistency related to names, groups, presets, ect. There are over 10k lines of code already. I'd say it should be ready around the end of November. Thanks for your support.

**Comment by jomixlaf** [comment]
Here is a sneak peek 🙃

<img width="650" height="1080" alt="Image" src="https://github.com/user-attachments/assets/4788a371-1da5-4944-ba53-6c28b2c52735" />

<img width="665" height="1110" alt="Image" src="https://github.com/user-attachments/assets/681396f3-4197-44dd-bd33-b503b3fb8628" />

<img width="643" height="1086" alt="Image" src="https://github.com/user-attachments/assets/a7fcfe1a-04b2-4ec3-a3fb-e41ae3345025" />

**Comment by devBenLA** [comment]
Hi, any update on this module?

**Comment by jomixlaf** [comment]
The module has been pushed to the official repo waiting for appouval to be part of the  offline package. This is on Bitfocus side :) Until then, you can clone the repo and install it on your machine; set up a dev folder, and it should work right away!

**Comment by devBenLA** [comment]
> The module has been pushed to the official repo waiting for appouval to be part of the offline package. This is on Bitfocus side :) Until then, you can clone the repo and install it on your machine; set up a dev folder, and it should work right away!

Great! I'll give it a try!

**Comment by devBenLA** [comment]
I've cloned the repo to my dev folder and can find the module inside companion now. However I can not set up a connection; it states "No config data loaded" and "Connection not found or not running". It doesn't even display the fields for IP and such.
OSX 14.8.2 running Companion 4.2.1.
Have you encountered this before?
I'll have a look to see if I can figure it out.

##Edit:
Turns out it was an issue with my local machine, which let me clone the repo, but not build it. Solved now.

**Comment by jomixlaf** [comment]
Fantastic, Bitfocus team asked me to change some thing prior to release and I've added some features and fixed bug as well so a new version is coming later beginning of 2026.

**Comment by briansin** [comment]
This is amazing, thank you so much @jomixlaf I cannot wait for this module to be officially released, it will solve so many problems in my home studio!

**Comment by jomixlaf** [comment]
Do you mind sharing your problem you have in your studio related to the Galaxy? What is your use case? 

Thanks

**Comment by briansin** [comment]
i need to give totally non technical people a way of muting certain outputs from galaxy (it’s a complex setup, but certain parts need to be controlled by people who know nothing about the system).  The ability to have a stream deck module button on a wall be the thing that mutes and unmutes these galaxy outputs is huge for me!

**Comment by jomixlaf** [comment]
Indeed, it seems like a very good use case for you. You'll be able to multi-select the input or output of your choice instead of duplicating the actions. I don't know when it will be released to the wild, but you can already clone the repo and use it right away on your machine. in the companion setting you point to a dev folder of your choice and put the galaxy folder in it, it should then be available as a module to use. Have fun

<img width="852" height="673" alt="Image" src="https://github.com/user-attachments/assets/a169bcc0-d1fa-4169-97a5-d4c679289231" />

**Comment by Phishy13** [comment]
I attempted to install your module by downloading the .tgz and by adding it to a dev folder, I managed to get it to show up in my connections but when I add it as a connection I get a "Connection not found or not running" error. Any idea why this is?

**Comment by jomixlaf** [comment]
Does it display the dropdown to choose either virtual or real Galaxy?

If you can post a print screen im happy to help

**Comment by Phishy13** [comment]
It displays this on each version I attempted to use.

<img width="914" height="629" alt="Image" src="https://github.com/user-attachments/assets/4904f017-6d21-4242-9ce4-82060bb7e659" />
<img width="914" height="629" alt="Image" src="https://github.com/user-attachments/assets/372a35d4-d146-4567-a4ab-3209ab6699be" />

**Comment by Phishy13** [comment]
Figured it out, it was user error of not running yarn companion-module-build so it wasn’t in the correct format.

**Comment by jomixlaf** [comment]
I've been very busy recently, we are version 1.4 but I am still waiting for Bitfocus to approve officially the module, it's been 3 months :(

**Comment by jbernard5** [comment]
I am trying to get this module to work and can't for the life of me succeed.   I have followed the steps and its just says Connection is not running

**Comment by jbernard5** [comment]
> Figured it out, it was user error of not running yarn companion-module-build so it wasn’t in the correct format.

How did you get it to work ?
